packages/http-response-serializer/__tests__/index.js

```diff
@@ -252,4 +252,26 @@ describe('📦  Middleware Http Response Serializer', () => {
       body: '{}'
     })
   })
+
+  test('It should return false when response body is falsey', async () => {
+    const handler = middy((event, context, cb) => {
+      cb(null, { statusCode: 200, body: false })
```

Comment:
If I got it right, this test case is not testing introduced changes in the detection of the type of response.  Response should be 
```
cb(null, false)
// or
cb(null, 0)
```
to test your change?

Reply:
This middleware will serialize the `body` - the issue occurred when `body` is `false`y

Reply:
Maybe I got this wrong, but this test would pass with a previous implementation too, right?

Reply:
I'll have a look @dkatavic